Userland/Libraries/LibWeb/Page/Frame.h

```diff
@@ -101,6 +103,12 @@ class Frame : public TreeNode<Frame> {
 
     void did_edit(Badge<EditEventHandler>);
 
+    void register_frame_nesting(URL url);
+    bool is_frame_nesting_allowed(URL url);
```

Comment:
Since this does not modify the `Frame` object, let's make it const-qualified.
Also since we are not passing ownership of the `URL`, let's pass it by const-reference:
```cpp
bool is_frame_nesting_allowed(URL const&) const;
```